plugins/sql-language/src/main/java/org/polypheny/db/sql/language/util/ReflectiveSqlOperatorTable.java

```diff
@@ -120,7 +120,7 @@ public List<Operator> getOperatorList() {
 
 
     /**
-     * Key for looking up operators. The name is stored in upper-case because we store case-insensitively, even in a case-sensitive session.
+     * Key for looking up operators. The name is stored in upper-case because we storeId case-insensitively, even in a case-sensitive session.
```

Comment:
This looks like a find and replace error

Reply:
Adjusted.